src/ui/public/modals/modal_overlay.js

```diff
@@ -5,16 +5,23 @@ import modalOverlayTemplate from './modal_overlay.html';
  * Appends the modal to the dom on instantiation, and removes it when destroy is called.
  */
 export class ModalOverlay {
-  constructor(modalElement) {
+  constructor(modalElement, scope) {
```

Comment:
I also agree that ESC traditionally maps to "Cancel" in basically any dialog that I can think of.

@Stacey-Gammon The reason I put this in the ModalOverlay rather than the ConfirmDialog itself was so that the functionality could be re-used for other dialog types when they are created. I figured that ESC->Cancel is universal enough that it would be re-used for any other dialog. If that's not desired then it is easy enough to move that code to the dialog itself.